KUBEVIRT_LOCAL_TESTING.md

```diff
@@ -8,7 +8,7 @@ Steps:
 
```

Comment:
the first sentence on the file is a bit out-of context. It is unclear to me what "the changes" is referring to. Can you rephrase it?
Do you mean something like "After making changes in a kubevirtci provider, you should test it locally before publishing it."  @dhiller ?

Reply:
> Do you mean something like "After making changes in a kubevirtci provider, you should test it locally before publishing it."

Yes i think this is the meaning, i will rephrase it, and lets see what Daniel thinks parallel to that.

Reply:
Updated, tried to keep both of the approaches of you and Daniel intact.